rotkehlchen/tests/unit/decoders/test_curve.py

```diff
@@ -804,6 +804,52 @@ def test_deposit_multiple_tokens(ethereum_transaction_decoder, ethereum_accounts
     assert events == expected_events
 
 
+@pytest.mark.vcr()
+@pytest.mark.parametrize('ethereum_accounts', [['0x66215D23B8A247C80c2D1B7beF4BefC2AB384bCE']])
+def test_gauge_vote(
+        ethereum_accounts,
+        ethereum_transaction_decoder,
+) -> None:
```

Comment:
do not break lines if there is no need

```suggestion
def test_gauge_vote(ethereum_accounts, ethereum_transaction_decoder) -> None:
```